src/main/python/hydra/lib/mmapi.py

```diff
@@ -105,6 +107,30 @@ def update_slaves(self):
             self.slavesHN[itm['hostname']] = itm
             self.slavesID[itm['id']] = itm
 
+    def get_health(self):
+        r = requests.get(self.myaddr + '/master/state')
+        if (r.status_code == 200):
+            return True
+        return False
+
+    def get_version(self):
+        # returns
+        # {"build_date":"2016-02-23 00:35:03","build_time":1456187703.0,"build_user":"root",
+        #  "git_sha":"864fe8eabd4a83b78ce9140c501908ee3cb90beb","git_tag":"0.27.1","version":"0.27.1"}
+        r = requests.get(self.myaddr + '/version')
+        if (r.status_code == 200):
+            return json.loads(r.content)
+        return None
```

Comment:
Do you do based on this return ? Better would be to raise here or raise based on return of the code is not what you expect.